AutomatedTesting/Gem/PythonTests/editor/test_Editor_Periodic.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
As per standarization, we should call these files TestSuite_Main.py, TestSuite_Periodic, etc

Reply:
Done.